catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/dataformats.properties

```diff
@@ -21,6 +21,7 @@ gson
 gzipDeflater
 hl7
 ical
+j8583
```

Comment:
Is this supposed to be the name of the library? Or the name of the format? Because the format/protocol is ISO8583; j8583 is the name of the Java library.

Reply:
yeah its something we decide, its not really as much visibile to end users - but using iso8583 is likely better

Reply:
renamed